.travis.yml

```diff
@@ -2,22 +2,11 @@
 language: ruby
 
 rvm:
-  - 1.9.3
-  - 2.1.0
+  - 2.1.9
   - 2.3.1
```

Comment:
I dont think they are using 2.3.1 yet and might skip it entirely for 2.4. Suggest removing this.